### ISSUE TITLE
Replace links -g with a Python script

### ISSUE THREAD

**Issue by gerryd** [FOSDEM/video#40]
Very low priority (as in: only after everything else works), but I'd like to replace the ugly hack that is running `links -g`  on the framebuffer as well as `update-statuspage.sh`. It shouldn't be too hard to write a small Python script with PyGame that produces something prettier than what we currently have.

**Comment by gerryd** [comment]
Sorry, I couldn't help myself.
